core/worker/src/main/java/com/cognifide/aet/worker/drivers/FirefoxWebDriverFactory.java

```diff
@@ -129,23 +135,17 @@ private FirefoxProfile getFirefoxProfile() throws IOException {
     return firefoxProfile;
   }
 
-  private AetFirefoxDriver getFirefoxDriver(FirefoxProfile fp, DesiredCapabilities capabilities)
+  private RemoteWebDriver getFirefoxDriver(DesiredCapabilities capabilities)
```

Comment:
We are declaring `RemoteWebDriver`, but are returning `FirefoxWebDriver`.
Could we return `WebDriver` interface here?

Reply:
ok